src/components/LoansYouMightLike/lymlContainer.vue

```diff
@@ -142,7 +142,7 @@ export default {
 			expMlLoanToLoanVersion: null,
 		};
 	},
-	inject: ['apollo'],
+	inject: ['apollo', 'cookieStore'],
```

Comment:
Is cookieStore needed in these components that don't use it?

Ok for now but maybe we can clean up later if it's not required in this context

Reply:
It's needed here because the apollo mixin uses it (to get the basket id).

Initially I tried to put the `inject` option in the mixin itself so that this line wouldn't be needed at all, but I ran into a bunch of warnings from Vue that apollo and cookieStore were not defined (I think it's related to the order we load the plugins, specifically vue-progressbar, which appears to create a Vue instance inside its install function). I decided not to focus on resolving that, as this can be cleaned up later like you said.

Reply:
Ahh I see. Thanks for the added info!